apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/task/ManifestEntryDownloadTaskIT.java

```diff
@@ -33,16 +34,23 @@
 public final class ManifestEntryDownloadTaskIT {
   private static final Logger LOGGER = LoggerFactory.getLogger(ManifestEntryDownloadTask.class);
 
+  /** only need a single instance of the S3 client. */
+  private static AmazonS3 s3Client;
   /** The S3 task manager. */
   private S3TaskManager s3TaskManager;
 
+  /** Sets the minio test container. */
+  @BeforeAll
+  public static void setupMinioS3Client() {
+    s3Client = S3Utilities.createS3Client(new ExtractionOptions("foo"));
+  }
+
```

Comment:
Is this the same `"foo"` that we might want a constant for?

Reply:
So are we saying that on anything that has a hack text, it has to be a constant??? For what purpose???